src/main/java/seedu/address/model/Model.java

```diff
@@ -70,9 +71,9 @@
     void setUser(Person person);
 
     /**
-     * @return Person in the model.
+     * @return ObservablePerson in the model.
      */
-    Person getUser();
+    ObservablePerson getUser();
```

Comment:
I get that user always refers to ObservablePerson but can we just name it observablePerson?

Reply:
This I'm not sure.
From the perspective of model does it know of the existence of `ObservablePerson`?

Reply:
Nvm it should. I will update.